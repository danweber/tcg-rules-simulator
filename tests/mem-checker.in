VERSION:2


#[Your Turn] While you have 0 or more memory, this Monster gets +80 DP.
#[Your Turn] While your opponent has 1 or more memory, this Monster gets +40 DP.
# etc

# I think this is right but I'm 

FIELD1:DW1-MemWatcher
FIELD2:DW1-MemWatcher
GAME:P1 T4 MAIN -6
DUMP:DP
ECHO: ---
GAME:P1 T4 MAIN -5
DUMP:DP
ECHO: ---
GAME:P1 T4 MAIN -4
DUMP:DP
ECHO: ---
GAME:P1 T4 MAIN -3
DUMP:DP
ECHO: -- huh
GAME:P1 T4 MAIN -2
DUMP:DP
ECHO: -----zzzz
GAME:P1 T4 MAIN -1
DUMP:DP
GAME:P1 T4 MAIN -0
DUMP:DP
GAME:P1 T4 MAIN 0
DUMP:DP
GAME:P1 T4 MAIN 1
DUMP:DP
GAME:P1 T4 MAIN 2
DUMP:DP
ECHO: --- 
GAME:P1 T4 MAIN 3
DUMP:DP
GAME:P1 T4 MAIN 4
DUMP:DP
GAME:P1 T4 MAIN 5
DUMP:DP
GAME:P1 T4 MAIN 6
DUMP:DP


EXIT

ECHO: -- play big boy --
CMD:PLAY BT14-Fenri Mem

GET-X-PLAYS
DUMP:HAND,DP

ECHO: -- use 1 cost --
CMD:PLAY BT2-DarknessClaw
STEP:AUTOANSWER:CAPTURE
DUMP:HAND,DP

EXIT


