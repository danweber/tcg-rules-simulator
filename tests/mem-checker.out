FIELD1:Xxx,DW1-Memwatcher,2000DP
FIELD2:Xxx,DW1-Memwatcher,2000DP
GAME:P1 T4 MAIN -6
ECHO: ---
FIELD1:Xxx,DW1-Memwatcher,2000DP
FIELD2:Xxx,DW1-Memwatcher,2000DP
GAME:P1 T4 MAIN -5
ECHO: ---
FIELD1:Xxx,DW1-Memwatcher,2001DP
FIELD2:Xxx,DW1-Memwatcher,1600DP
GAME:P1 T4 MAIN -4
ECHO: ---
FIELD1:Xxx,DW1-Memwatcher,2003DP
FIELD2:Xxx,DW1-Memwatcher,1400DP
GAME:P1 T4 MAIN -3
ECHO: -- huh
FIELD1:Xxx,DW1-Memwatcher,2007DP
FIELD2:Xxx,DW1-Memwatcher,1300DP
GAME:P1 T4 MAIN -2
ECHO: -----zzzz
FIELD1:Xxx,DW1-Memwatcher,2017DP
FIELD2:Xxx,DW1-Memwatcher,1260DP
GAME:P1 T4 MAIN -1
FIELD1:Xxx,DW1-Memwatcher,2037DP
FIELD2:Xxx,DW1-Memwatcher,1240DP
GAME:P1 T4 MAIN 0
FIELD1:Xxx,DW1-Memwatcher,2037DP
FIELD2:Xxx,DW1-Memwatcher,1240DP
GAME:P1 T4 MAIN 0
FIELD1:Xxx,DW1-Memwatcher,2077DP
FIELD2:Xxx,DW1-Memwatcher,1230DP
GAME:P1 T4 MAIN 1
FIELD1:Xxx,DW1-Memwatcher,2177DP
FIELD2:Xxx,DW1-Memwatcher,1226DP
GAME:P1 T4 MAIN 2
ECHO: ---
FIELD1:Xxx,DW1-Memwatcher,2377DP
FIELD2:Xxx,DW1-Memwatcher,1224DP
GAME:P1 T4 MAIN 3
FIELD1:Xxx,DW1-Memwatcher,2777DP
FIELD2:Xxx,DW1-Memwatcher,1223DP
GAME:P1 T4 MAIN 4
FIELD1:Xxx,DW1-Memwatcher,2777DP
FIELD2:Xxx,DW1-Memwatcher,1223DP
GAME:P1 T4 MAIN 5
FIELD1:Xxx,DW1-Memwatcher,2777DP
FIELD2:Xxx,DW1-Memwatcher,1223DP
GAME:P1 T4 MAIN 6
